client/src/components/dashboard/verify/Verify.tsx

```
@@ -0,0 +1,137 @@
+        setVerificationStatus(data.status);
+      } catch (error) {
+        console.error('Error verifying status:', error);
+      }
+    }
+    setActiveStep((prev) => prev + 1);
+  };
+
+  const handleBack = () => setActiveStep((prev) => prev - 1);
+
+  const handleFileChange = (event: React.ChangeEvent<HTMLInputElement>) => {
+    if (event.target.files && event.target.files[0]) {
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)